test/integration/puppetlabs/services/jruby/jruby_puppet_pool_int_test.clj

```diff
@@ -362,6 +362,35 @@
          (finally
            (jruby-testutils/return-instance jruby-service jruby-instance :settings-plumbed-test)))))))
 
+(deftest jruby-environment-vars-test
+  (testing "Make sure that the environment variables whitelisted in puppetserver.conf are being set"
+    (tk-testutils/with-app-with-config
+     app
+     [jruby/jruby-puppet-pooled-service
+      profiler/puppet-profiler-service
+      jruby-utils/jruby-pool-manager-service]
+     (jruby-testutils/jruby-puppet-tk-config
+      (jruby-testutils/jruby-puppet-config
+       {:ruby-load-path  jruby-testutils/ruby-load-path
+        :gem-home        jruby-testutils/gem-home
+        :master-conf-dir jruby-testutils/conf-dir
+        :master-code-dir jruby-testutils/code-dir
+        :master-var-dir  jruby-testutils/var-dir
+        :master-run-dir  jruby-testutils/run-dir
+        :master-log-dir  jruby-testutils/log-dir
+        :environment-vars jruby-testutils/environment-vars}))
```

Comment:
Both @cprice404 and I were thinking it would be good to have some sort of integration-level test for this functionality which shows that an environment variable expressed in HOCON can be extracted and delivered all the way through to the running JRuby container.  I think this is a good test as it is but I think we could tweak it a bit to cover the HOCON configuration bit as well.  Maybe we could do something like:

1) Use the Trapperkeeper `load-config` function to parse a HOCON configuration file back into a map.

See https://github.com/puppetlabs/puppetserver/blob/5345de0b27fc34499ab07f04169b3f0ab8435197/test/integration/puppetlabs/puppetserver/bootstrap_testutils.clj#L39-L50 for an example of how this is done behind some of our test helpers.

In this case, I'm thinking that the HOCON file on disk might have a section like:

```hocon
jruby-puppet: {
    ...
    environment-vars: { "FOO": #{PATH} }
}
```

2) Merge the result from the `load-config` with what you're already passing through to `jruby-puppet-config`. 

I'd be fine with just loading up a HOCON file with just that one `environment-vars` setting in it.  Just exercising HOCON to see that the environment variable expansion is happening as expected seems like it would be good enough.

3) Validate that the value for `FOO` stored in the `jruby-env` is equal to the value that you'd get when calling:

```clj
(System/getEnv "PATH")
```

Make sense?